authorization/api.go

```diff
@@ -6,7 +6,9 @@ import (
 	"encoding/pem"
 	"net/http"
 
-	"github.com/docker/go-plugins-helpers/sdk"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Scalingo/go-plugins-helpers/sdk"
```

Comment:
This change is needed so that the specs use the function `createPluginSpecDirUnix` we updated. Our version of the function use an environment variable `PLUGIN_SPEC_DIR` in the specs.